### PR TITLE
Add instructions for installing pact for tests

### PR DESCRIPTION
**Pull request by johnmcollier**
Provides instructions for installing the pact tools for the tests, otherwise the controller tests don't run.

### CODE REVIEW

**Review by maysunfaisal — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -44,6 +45,16 @@ The image being pushed can again be modified using the environment variable:
 IMG=quay.io/user/hasoperator:next make docker-push
 ```
 
+## Installing Pact Tools
+
+The Pact tests in the controller package require pact tooling to be installed and on your path. Follow these instructions to do so:
+
+1. Change directory to an appropriate folder (e.g. `/usr/local`)
+2. Run `curl -fsSL https://raw.githubusercontent.com/pact-foundation/pact-ruby-standalone/master/install.sh | bash`
+3. Add the pact tools' bin folder (e.g. `/usr/local/pact/bin`) to your path to your shell PATH. Might need to ensure the `bin/` folder has sufficient permissions
```
Comment (michael-valdron) [inline README.md:54]:
```suggestion
3. Add the pact tools' bin folder (e.g. `/usr/local/pact/bin`) to your path to your shell PATH. Ensure all binary files within the `bin/` folder has executable permissions
```

**Review by michael-valdron — CHANGES_REQUESTED** [review]

**Review by michael-valdron — APPROVED** [review]
@johnmcollier My change request is minor, let know if you decide to make the change and I'll re-approve.

**Comment by johnmcollier** [comment]
@michael-valdron updated

**Review by michael-valdron — APPROVED** [review]
/lgtm

**Pull request merged**